### DISCUSSION TITLE
PIC32MX254F128DT-I/ML vs PIC32MX250F128D-I/ML

### DISCUSSION THREAD

**Discussion by JacobChrist** [chipKIT32/chipKIT-core#474] [Q&A] [answered]
Anyone know the difference between PIC32MX254F128DT-I/ML vs PIC32MX250F128D-I/ML.  Digikey has about 1,600 of the PIC32MX254F128DT-I/ML in stock while the PIC32MX2x0FxxxD-I/ML are pretty much wiped out.  Digikey points to the datasheet for the PIC32MX1XX/2XX 28/36/44-PIN FAMILY but I can't find anything in the datasheet that indicates what the 4 or the DT designates in the part number.

Jacob

**Comment by majenkotech** [comment, marked as answer]
If it's anything like the 274 it has a very different oscillator including
the ability to sync the internal oscillator with the clock recovered from
the usb connection. Very groovy stuff.

I think you need the 1xx/2xx XLP datasheet.

Late here so I can't look right now...

**Reply by JacobChrist** [reply]
@majenkotech I do see that it runs up at 72MHz on Digikey, they must be pointing to the wrong datasheet.  I did find this one based on your suggestion:

https://ww1.microchip.com/downloads/en/DeviceDoc/PIC32MX1XX2XX_XLP_Family_Datasheet_DS60001404E.pdf

Jacob
